Lib/typing.py

```diff
@@ -989,10 +989,21 @@ def _allow_reckless_class_cheks():
         return True
 
 
-_PROTO_WHITELIST = ['Callable', 'Awaitable',
-                    'Iterable', 'Iterator', 'AsyncIterable', 'AsyncIterator',
-                    'Hashable', 'Sized', 'Container', 'Collection', 'Reversible',
-                    'ContextManager', 'AsyncContextManager']
+_PROTO_WHITELIST = {
+    'Callable': 'collections.abc',
```

Comment:
I'd like to write code in this way.
`'Callable': collections.abc.__name__,`
It can catch typo error.

Reply:
Alternatively I would just avoid the repetition by making `_PROTO_WHITELIST` a dictionary mapping module to set of names in it. So that the code below would be
```python
    base.__module__ in _PROTO_WHITELIST and
    base.__name__ in _PROTO_WHITELIST[base.__module__]
```